src/OrchardCore.Modules/OrchardCore.ContentFields/Fields/ContentPickerField.cs

```diff
@@ -4,6 +4,6 @@ namespace OrchardCore.ContentFields.Fields
 {
     public class ContentPickerField : ContentField
     {
-        public string[] ContentItemIds { get; set; }
+        public string[] ContentItemIds { get; set; } = new string[0];
```

Comment:
```suggestion
        public string[] ContentItemIds { get; set; } = Array.Empty<string>();
```

Reply:
Updated

Reply:
Just waiting for the CI to go green, but it's queued